utils/model_uploader/sparse_model_autotracing.py

```diff
@@ -0,0 +1,465 @@
+    """
+    folder_path = (
+        TORCHSCRIPT_FOLDER_PATH
+        if model_format == TORCH_SCRIPT_FORMAT
+        else ONNX_FOLDER_PATH
+    )
+
+    # 1.) Initiate a sparse encoding model class object
+    try:
+        pre_trained_model = SparseEncodingModel(
+            model_id=model_id, folder_path=folder_path, overwrite=True
+        )
+    except Exception as e:
+        assert (
+            False
+        ), f"Raised Exception in tracing {model_format} model\
+                             during initiating a sparse encoding model class object: {e}"
+
+    # 2.) Save the model in the specified format
+
+    try:
+        if model_format == TORCH_SCRIPT_FORMAT:
+            model_path = pre_trained_model.save_as_pt(
+                model_id=model_id,
+                sentences=TEST_SENTENCES,
+                add_apache_license=True,
+            )
+        else:
+            model_path = pre_trained_model.save_as_onnx(
+                model_id=model_id, add_apache_license=True
+            )
+    except Exception as e:
+        assert False, f"Raised Exception during saving model as {model_format}: {e}"
+
+    # 3.) Create a model config json file
+    try:
+        model_config_path = pre_trained_model.make_model_config_json(
+            version_number=model_version,
+            model_format=model_format,
+            description=model_description,
+        )
+    except Exception as e:
+        assert (
+            False
+        ), f"Raised Exception during making model config file for {model_format} model: {e}"
+
+    # 4.) Preview model config
+    print(f"\n+++++ {model_format} Model Config +++++\n")
+    with open(model_config_path, "r") as f:
+        model_config = json.load(f)
+        print(json.dumps(model_config, indent=4))
+    print("\n+++++++++++++++++++++++++++++++++++++++\n")
+
+    # 5.) Return model_path & model_config_path for model registration
+    return model_path, model_config_path
+
+
+def register_and_deploy_sparse_encoding_model(
+    ml_client: "MLCommonClient",
+    model_path: str,
+    model_config_path: str,
+    model_format: str,
+    texts: list[str],
+) -> list:
+    encoding_datas = None
+    model_id = autotracing_utils.register_and_deploy_model(
+        ml_client, model_format, model_path, model_config_path
+    )
+    autotracing_utils.check_model_status(ml_client, model_id, model_format)
+    try:
+        encoding_output = ml_client.generate_sparse_encoding(model_id, texts)
+
+        encoding_datas = [
+            encoding_output["inference_results"][i]["output"][0]["dataAsMap"][
+                "response"
+            ][0]
+            for i in range(len(texts))
+        ]
+
+    except Exception as e:
+        assert (
+            False
+        ), f"Raised Exception in generating sparse encoding with {model_format} model: {e}"
+    autotracing_utils.undeploy_model(ml_client, model_id, model_format)
+    autotracing_utils.delete_model(ml_client, model_id, model_format)
```

Comment:
Why not add some assertions like register_and_deploy_sentence_transformer_model in model_autotracing.py?